tests/xtro-sharpie/watchOS-AVFoundation.ignore

```diff
@@ -17,3 +17,10 @@
 !unknown-field! AVAudioUnitTypeOfflineEffect bound
 !unknown-field! AVAudioUnitTypeOutput bound
 !unknown-field! AVAudioUnitTypePanner bound
+
+# added the class to expose a filed and match the other APIs.
+!unknown-type! AVCaptureDevice bound
+!unknown-type! AVContentKeyRequest bound
```

Comment:
^^ happy to see xtro caught them (and that there's no more than the two I spotted)